packages/proto-ble/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "proto-ble",
```

name should be `"@helium/proto-ble"`

```suggestion
  "name": "@helium/proto-ble",
```